modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/SourceResponse.java

```diff
@@ -248,8 +248,9 @@ private void calculateContentlengthForChunckDisabledResponse(MessageContext resp
             String contentType = headers.get(HTTP.CONTENT_TYPE) != null ?
                     headers.get(HTTP.CONTENT_TYPE).toString() : null;
             // Stream should be preserved to support disable chunking for SOAP based responses. This checks
-            // whether chunking is disabled and response Content-Type is SOAP.
-            boolean shouldPreserve = isChunkingDisabled && isSOAPContentType(contentType);
+            // whether chunking is disabled and response Content-Type is SOAP or FORCE_HTTP_1.0 is true.
+            boolean shouldPreserve =
```

Comment:
What about a variable name like shouldPreserveChunking? forceChunking?

Reply:
In here we are preserving the stream (if chunking is disabled and response Content-Type is SOAP, the stream is preserved).

Reply:
Then a name like preserveStream? We need to add what to be preserved in the variable name.

Reply:
Changed the name into preserveStream.